website/docs/cheatsheets/python/v1/cheatsheet.md

```diff
@@ -21,9 +21,9 @@ ws = Workspace.from_config()
 The workspace object is the fundamental handle on your Azure ML assets and is used
 throughout (often simply referred to by `ws`).
 
-For more details: [Workspaces](workspace)
```

Comment:
On live site, it thinks the link is "https://azure.github.io/azureml-cheatsheets/docs/cheatsheets/python/v1/cheatsheet/workspace" whereas it should be "https://azure.github.io/azureml-cheatsheets/docs/cheatsheets/python/v1/workspace".